src/api/services/login-service.ts

```diff
@@ -22,58 +19,59 @@ import {Subject} from 'rxjs/Subject';
 @Injectable()
 export class LoginService extends CoreWebService {
 
-    private _loginUser$: Subject<User> = new Subject<User>();
-    private changePasswordURL: string;
+    private _auth$: Subject<Auth> = new Subject<Auth>();
+    private _auth: Auth;
+    private _loginAsUsersList$: Subject<User[]>;
     private country: string = '';
-    private _isLoginAs$: Subject<boolean> = new Subject<boolean>();
     private lang: string = '';
-    private loginAsUrl: string;
-    private logoutAsUrl: string;
     private loginAsUserList: User[];
-    private logoutURL: string;
-    private recoverPasswordURL: string;
-    private serverInfoURL: string;
-    private user: User;
-    private userAuthURL: string;
-    private userListUrl: string;
-    private _loginAsUsers$: Subject<User[]>;
+    private urls: any;
 
-    constructor(apiRoot: ApiRoot, http: Http, public coreWebService: CoreWebService, private router: Router,
-                @Inject('dotcmsConfig') private dotcmsConfig: DotcmsConfig, private  formatDate: FormatDate,
-                private appConfigurationService: AppConfigurationService) {
+    constructor(apiRoot: ApiRoot, http: Http, public coreWebService: CoreWebService, private router: Router) {
         super(apiRoot, http);
 
-        this._isLoginAs$ = <Subject<boolean>>new Subject();
-        this._loginAsUsers$ = <Subject<User[]>>new Subject();
-        this.changePasswordURL = `${apiRoot.baseUrl}api/v1/changePassword`;
-        this.loginAsUrl = `${apiRoot.baseUrl}api/v1/users/loginas/userid`;
+        this._loginAsUsersList$ = <Subject<User[]>>new Subject();
         this.loginAsUserList = [];
-        this.logoutAsUrl = `${apiRoot.baseUrl}api/v1/users/logoutas`;
-        this.logoutURL = `${apiRoot.baseUrl}api/v1/logout`;
-        this.recoverPasswordURL = `${apiRoot.baseUrl}api/v1/forgotpassword`;
-        this.serverInfoURL = `${apiRoot.baseUrl}api/v1/loginform`;
-        this.userAuthURL = `${apiRoot.baseUrl}api/v1/authentication`;
-        this.userListUrl = `${apiRoot.baseUrl}api/v1/users/loginAsData`;
+        this.urls = {
```

Comment:
I think that we should call the requestView method like this better:

requestView({
   url: 'v1/authentication',
  ...
})

I did a change in CoreWebService that allow it